### PR TITLE
Hyundai: Add Car Port for 2022 Hyundai Santa Fe

### PR DESCRIPTION
**Pull request by sunnyhaibin**
**Checklist**
- [x] added to README, RELEASES, CARS
- [x] test route added to [test_routes.py](https://github.com/sunnyhaibin/openpilot-1/blob/ae5590654a667248e97de37fe39522dc69dc9b58/selfdrive/test/test_routes.py#L71)
- [x] route with openpilot: `bf43d9df2b660eb0|2021-09-23--14-16-37`
- [x] Harness Type: [Hyundai L](https://github.com/commaai/openpilot/wiki/Hyundai-Kia-Genesis#harness-guide)

Thanks to community Santa Fe owner ER3#3623 (Discord ID).

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Version 0.8.10 (2021-XX-XX)
  * Škoda Karoq 2019 support thanks to jyoung8607!
  * Volkswagen Arteon 2021 support thanks to jyoung8607!
  * Volkswagen California 2021 support thanks to jyoung8607!
+ * Hyundai Santa Fe 2022 support thanks to sunnyhaibin!
```
Comment (adeebshihadeh) [inline RELEASES.md:7]: can you move this to the top? car ports are usually ordered alphabetically

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -105,6 +106,9 @@ class Buttons:
   {
     67: 8, 68: 8, 80: 4, 160: 8, 161: 8, 272: 8, 288: 4, 339: 8, 356: 8, 357: 8, 399: 8, 544: 8, 608: 8, 672: 8, 688: 5, 704: 1, 790: 8, 809: 8, 848: 8, 880: 8, 898: 8, 900: 8, 901: 8, 904: 8, 1056: 8, 1064: 8, 1065: 8, 1072: 8, 1075: 8, 1087: 8, 1088: 8, 1151: 8, 1200: 8, 1201: 8, 1232: 4, 1264: 8, 1265: 8, 1266: 8, 1296: 8, 1306: 8, 1312: 8, 1322: 8, 1331: 8, 1332: 8, 1333: 8, 1348: 8, 1349: 8, 1369: 8, 1370: 8, 1371: 8, 1407: 8, 1415: 8, 1419: 8, 1440: 8, 1442: 4, 1461: 8, 1470: 8
   }],
+  CAR.SANTA_FE_2022: [{
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:109]: remove this, we don't use can fingerprints anymore

File: RELEASES.md (hunk 1 above)
Comment (sunnyhaibin) [reply RELEASES.md]: Done.

File: selfdrive/car/hyundai/values.py (hunk 2 above)
Comment (sunnyhaibin) [reply selfdrive/car/hyundai/values.py]: Done.

**Pull request merged**